### PR TITLE
For #2741

### PR DESCRIPTION
**Pull request by boek**

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt [hunk 1]
```diff
@@ -739,4 +742,11 @@ object TelemetryWrapper {
     fun addSearchEngineLearnMoreEvent() {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.ADD_SEARCH_ENGINE_LEARN_MORE).queue()
     }
+
+    private fun isDistributedDevice(): Boolean {
```
Comment (pocmo) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: tiny nit: We may not want to disable this for all distribution, so I wonder if we could make the method name more explicit... something like isDeviceWithTelemetryDisabled()?

Comment (boek) [reply app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: Sounds good to me

**Comment by pocmo** [comment]
@boek Are you taking care of merging this into master too?

**Comment by boek** [comment]
Yep! I'll put it into master and 6.0 release branch

**Pull request merged**